shell/platform/android/platform_view_android.cc

```diff
@@ -30,21 +30,36 @@ PlatformViewAndroid::PlatformViewAndroid(
          "rendering.";
 }
 
+PlatformViewAndroid::PlatformViewAndroid(
```

Comment:
I would be more comfortable with this being a separate subclass of PlatformView. Though you have added a null checks to the surface on each call site, there are also other callbacks that make no sense in a background execution context. Stuff like the first frame callback and such.

We can iterate on this in a subsequent patch however. I don't think this is necessarily blocking.

Reply:
That sounds like a plan!